src/main/java/com/aws/greengrass/device/ClientDevicesAuthService.java

```diff
@@ -199,4 +215,9 @@ private void uploadCoreDeviceCAs(List<String> certificatePemList) {
                     String.format("Failed to put core %s CA certificates to cloud", thingName), e);
         }
     }
+
+    // Check if the changed node belongs to the topics.
+    private boolean isParentTopic(Node changedNode, Node topics) {
```

Comment:
you don't need this.

`childOf()` already exists

Reply:
Fixed